inc/saiport.h

```diff
@@ -794,6 +794,32 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_EGRESS_MACSEC_ACL,
 
+    /**
+     * @brief Port bind point for ingress MACsec port object
```

Comment:
SAI_MACSEC_PORT_ATTR_PORT_ID is removed from sai_macsec_port_attr_t so there is no cross reference of gearbox port from MACsec port. It is required for SA create. 
What is the significance of it adding here?

Reply:
sai_macsec_port is associated to a sai_port by SAI_PORT_ATTR_INGRESS_MACSEC_PORT or SAI_PORT_ATTR_EGRESS_MACSEC_PORT.  SAI does not allow a reverse pointer.

Reply:
Please ignore previous comment.  I can restore SAI_MACSEC_PORT_ATTR_PORT_ID if I replace SAI_PORT_ATTR_INGRESS_MACSEC_PORT and SAI_PORT_ATTR_EGRESS_MACSEC_PORT with SAI_PORT_ATTR_MACSEC_PORT_LIST.  Will do so.